content/news/2025/01/15/pybamm-25.1.md

```diff
@@ -0,0 +1,56 @@
+
+The full list of changes can be found in the [CHANGELOG](https://pybamm.org/changelog/) file, but here we provide a deeper overview of the main features of this release.
+
+## Increased compatibility of submodels
+_Implemented by [Alec Bills (Ionworks)](https://github.com/aabills) and [Simon O'Kane](https://github.com/DrSOKane)_
+
+Several submodels have been updated to enhance the compatibility with other submodels. The new compatibilities are:
+
+* SEI models with particle size distributions.
+* Composite electrodes with particle size distributions.
+* Porosity change with composite electrodes.
+
+This means that almost all submodels are cross-compatible!
+
+## SEI models are now for a single layer only
+_Implemented by [Valentin Sulzer (Ionworks)](https://github.com/valentinsulzer)_
+
+All SEI models in PyBaMM are now defined for a single layer SEI, and the double layer SEI models have been removed. Note that by double layer we mean an SEI with inner and outer SEI layers, not the interface capacitance double layer. This change should make SEI models easier to understand and compare.
+
+## Two new SEI models
+_Implemented by [Kawa Manmi (University of Warwick)](https://github.com/kawaMANMI)_
+
+Two new SEI models have been added to PyBaMM: the `"tunnelling limited"` model (from [Tang et al (2012)](https://iopscience.iop.org/article/10.1149/2.025211jes)) and the `"VonKolzenberg2020"` model (from [Von Kolzenberg et al (2020)](https://chemistry-europe.onlinelibrary.wiley.com/doi/full/10.1002/cssc.202000867)). These models can be enabled by passing the relevant `"SEI"` in the model options dictionary. Note that additional parameters are required for this model, and the parameter set dictionary needs to be updated manually.
+
+## Improved `search` method in PyBaMM dictionaries
+_Implemented by [Medha Bhardwaj (IIIT Lucknow)](https://github.com/medha-14)_
+
+The `search` method in PyBaMM dictionaries, for example in a `ParameterValues` or a `model.variables` dictionary, has been enhanced so it can take lists of strings. This allows for more flexibility when searching for variables or parameters in a dictionary. For example, if we want to search for all variables related to potentials in the negative electrode, we can now do
+
+```python3
+model.variables.search(["negative", "potential"])
+```
+
+## Geometric parameters as input parameters
+_Implemented by [Alec Bills (Ionworks)](https://github.com/aabills)_
+PyBaMM now includes `SymbolicUniform1DSubMesh`, which allows for the definition of geometric parameters as input parameters without need to rebuild the model. The mesh is internally defined to be defined between 0 and 1, and the discretised operators take care of the contribution of the domain lengths.
+
+This mesh does not work straight out of the box with default models, but it can be used in custom-built models.
+
+## Standalone package for C/C++ solvers
```

Comment:
This could potentially also include the updates to conda distribution (renaming the heading to something like "Improved distribution and packaging on PyPI and conda-forge") -
1. `noarch` distribution (similar to the pure Python wheel)
2. a lite `pybamm-base` package (and the `pybamm` package with all optional dependencies, including JAX)

Reply:
I don't know much about conda forge so I am not following the developments. Happy to include, can you suggest a change here and I will commit it?